JapaneseFilter/sections/whitelist.txt

```diff
@@ -5,6 +5,8 @@
 ! Bad:  @@||example.org^$stealth (should be in AdGuard Base - whitelist_stealth.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/115636
+netgeek.biz#@#div#content div[id*="-"][class*="%e"]:not([itemtype])
```

Comment:
Better add `:not(.post)` to the blocking rule. Related to https://github.com/AdguardTeam/AdguardFilters/commit/0fde18ec39331966df45df161fab8a489e19ed22
@Yuki2718 Is not the rule to dangerous?

Reply:
Yeah, maybe these rules should be removed, in the end they are only for placeholder/leftover on some Japanese blogs. I'll lator check those blogs to see if there can be safer alternatives.

Reply:
@jellizaveta Remove thsese:
```
##article[class^="post-"] div[class*="article-"] div[id*="-"][class*="%e"]:not([itemtype])
##div#content div[id*="-"][class*="%e"]:not([itemtype])
##[id][class*="post-"] [class*="-content"] div[id*="-"][class*="%e"]:not([itemtype])
```